### PR TITLE
CDP Build Steps

### PR DESCRIPTION
**Pull request by njuettner**
- CDP Steps for Kube-Ingress-Controller on AWS

### CODE REVIEW

File: delivery.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        export OPERATOR_TOP_DIR=$GOPATH/src/github.com/zalando-incubator
+        mkdir -p $OPERATOR_TOP_DIR
+        ln -s $(pwd) $OPERATOR_TOP_DIR/kube-ingress-aws-controller
+    - desc: 'Build & push docker image'
+      cmd: |
+        export PATH=$PATH:$HOME/go/bin
+        IS_PR_BUILD=${CDP_PULL_REQUEST_NUMBER+"true"}
+        if [[ ${CDP_TARGET_BRANCH} == "master" && ${IS_PR_BUILD} != "true" ]]
+        then
+          make build.docker build.push
```
Comment (mikkeloscar) [inline delivery.yaml]: `make build.push` is enough

**Comment by njuettner** [comment]
👍

File: delivery.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        apt-get install -y golang-go
+    - desc: 'Install Docker'
+      cmd: |
+        curl -sSL https://get.docker.com/ | sh
+    - desc: 'Symlink sources into the GOPATH'
+      cmd: |
+        export GOPATH=$HOME/go
+        export OPERATOR_TOP_DIR=$GOPATH/src/github.com/zalando-incubator
+        mkdir -p $OPERATOR_TOP_DIR
+        ln -s $(pwd) $OPERATOR_TOP_DIR/kube-ingress-aws-controller
```
Comment (mikkeloscar) [inline delivery.yaml]: nit: maybe also change the `OPERATOR_TOP_DIR` to another name, it's maybe a bit confusing why it's called this way.

**Comment by njuettner** [comment]
👍

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**